api/v1beta1/cloudstackaffinitygroup_conversion.go

```diff
@@ -30,3 +33,44 @@ func (dst *CloudStackAffinityGroup) ConvertFrom(srcRaw conversion.Hub) error { /
 	src := srcRaw.(*v1beta2.CloudStackAffinityGroup)
 	return Convert_v1beta2_CloudStackAffinityGroup_To_v1beta1_CloudStackAffinityGroup(src, dst, nil)
 }
+
+//nolint:golint,revive,stylecheck
```

Comment:
Why are we skipping linting here? I'm probably fine with it, I'd just like to understand.